helm-chart/renku/templates/authz/service.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "renku.fullname" . }}-authz
+  labels:
+    app: renku-authz
+    chart: {{ template "renku.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  type: ClusterIP
+  ports:
+    - port: 8443
+      targetPort: http
+      protocol: TCP
+      name: http
+    - port: 50051
+      targetPort: grpc
+      protocol: TCP
+      name: grpc
```

Comment:
We need a port for the metrics too, as well as the prometheus annotations

Reply:
Ok after the discussion in the weekly sync meeting it is clear to me that I will be doing this. So I will do this in a separate followup PR.